JudoPayDotNet/Http/Connection.cs

```diff
@@ -23,7 +23,7 @@ public class Connection
     {
         private readonly IHttpClient _httpClient;
 
-        private readonly Uri _baseAddress;
+        public readonly Uri BaseAddress;
```

Comment:
same here, i don't see why this would need to be public

Reply:
I was unsure whether someone debugging would ever want to know the URL they are processing against. However this sounds like a bit of an edge case so I have made it internal